guide/yaml/blueprinting-tips.md

```diff
@@ -22,28 +35,30 @@ of artifacts (e.g. RPMs, zip files, etc).
 
 Options for speeding up provisioning include those below.
 
-### Deploying to the "localhost" location
+#### Deploying to Bring Your Own Nodes (BYON)
 
-This is fast and simple, but has some obvious downsides:
+A BYON location can be defined, which avoids the time required to provision VMs. This is fast,
+but has the downside that artifacts installed during a previous run can interfere with subsequent 
+runs.
 
-* Artifacts are installed directly on your desktop/server.
+A variant of this is to use Vagrant (e.g. with VirtualBox) to create VMs on your local machine,
```

Comment:
Link out to [the distributed vagrant config](https://github.com/apache/brooklyn-dist/tree/master/vagrant)?